_dashboards/visbuilder.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: VisBuilder
+nav_order: 8
+---
+
+# VisBuilder
+
+VisBuilder is an experimental feature and shouldn't be used in a production environment. For updates on its progress, or if you want to leave feedback that helps improve the feature, see the [git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2280).
+{: .warning}
+
+You can use the VisBuilder visualization type in OpenSearch Dashboards to create data visualizations easily using a drag-and-drop gesture. With VisBuilder you have:
+
+* an immediate view of your data without the need to preselect the visualization output
```

Comment:
Bullet points should always be initially capitalized, unless the first word is part of a lowercase proper noun. If the bullet point completes the introductory stem, it takes terminating punctuation.